cep-XXXX.md

```diff
@@ -0,0 +1,568 @@
+
+## Specification
+
+### Add a new top-level key `packages.whl` to list wheels in a channel
+
+According to the current draft schema for [repodata.json][repodata-schema], repodata consists of five top
+level keys:
+
+- repodata_version
+- info
+- packages
+- packages.conda
+- removed
+- signatures
+
+This CEP proposes the addition of a new `packages.whl` section to account for the wheel format. This key points to a mapping that MUST contain [repodata record][repodata-record-schema] objects. The key of this mapping MUST be the wheel file name, and the contents of the package record MUST contain metadata related to that wheel.
+
+### Record structure and naming convention
+
+To support sparse repodata processing and maintain compatibility with conda's existing infrastructure, wheel records MUST use conda-style naming conventions rather than wheel filenames as keys. This allows solvers to efficiently filter packages by name without parsing the full package record.
+
+### Key naming requirements
+
+The key for each entry in `packages.whl` must follow this format:
+
+```
+{name}-{version}
+```
+
+Examples:
+
+- `httpx-0.28.1`
+- `typing_extensions-4.15.0`
+- `lazy-loader-0.4`
+
+### Why wheel names may differ from conda names
+
+Several factors can cause wheel names to differ from conda-style names:
+
+1. Name normalization: PyPI normalizes dashes to underscores (PEP 503)
+  - Example: `lazy-loader` (conda-forge) vs `lazy_loader` (PyPI wheel)
+2. Python-specific clarification: PyPI packages are implicitly Python libraries
+  - Example: `authzed-py` (conda-forge) vs `authzed` (PyPI)
+3. Variant differences: Conda may offer multiple variants with different dependencies
+  - Example: `seaborn-base` (conda-forge) vs `seaborn` (PyPI)
+4. Cross-channel naming: Different conda channels may use different names
+  - Example: `pyperformance` (conda-forge) vs `performance` (main)
+
+### Naming standard
+
+When there are naming differences between channels, wheel records MUST use the conda-forge package name as the standard. This choice is made because:
+
+- conda-forge is the largest community channel with the most packages
+- conda-forge naming conventions are well-established and documented
+- Using a single standard prevents ambiguity and tooling complexity
+- Channel operators adding wheels to repodata MUST determine the appropriate conda-style name by:
+  - Checking if the package already exists in conda-forge and using that exact name
+  - If not in conda-forge, applying conda naming conventions per [CEP 26 - Identifying Packages and Channels in the conda Ecosystem][cep-26]
+  - Documenting any naming mappings for their channel
+
+### Wheel download URLs
+
+This CEP introduces a new optional `artifact_url` field in package records to specify download locations for individual packages.
+
+> Note for this draft: The `artifact_url` field could also be added as a separate CEP to allow it for other record types.
```

Comment:
I think that would actually be a good idea to avoid asymmetries in package record specifications. Either that or we explicitly mention this new field is for all package record types.